src/udev/udev-rules.c

```diff
@@ -29,6 +30,7 @@
 #include "udev-builtin.h"
 #include "udev-event.h"
 #include "udev-rules.h"
+#include "udev-util.h"
```

Comment:
This is also included transitively so this could in theory be dropped.

Reply:
I think we shouldn't be relying on transitivity of includes and always be explicit.